### ISSUE TITLE
Add `Django 5.0` compatibility.

### ISSUE THREAD

**Issue by fabiocaccamo** [spookylukey/django-paypal#286]
Hi @spookylukey,
I have not debugged any aspect of the package, but this error is certain:

```python
  File ".../site-packages/django/forms/forms.py", line 339, in _clean_fields
    value = field.clean(value)
            ^^^^^^^^^^^^^^^^^^
  File ".../site-packages/django/forms/fields.py", line 204, in clean
    value = self.to_python(value)
            ^^^^^^^^^^^^^^^^^^^^^
  File ".../site-packages/paypal/standard/forms.py", line 91, in to_python
    dt = timezone.make_naive(dt, timezone=timezone.utc)
                                          ^^^^^^^^^^^^
AttributeError: module 'django.utils.timezone' has no attribute 'utc'
```

**Comment by spookylukey** [comment]
This was fixed in version 2.1

**Comment by fabiocaccamo** [comment]
@spookylukey I just updated the library, thank you very much!
